docs/data-sources/flow_milestone.md

```diff
@@ -3,12 +3,12 @@
 page_title: "genesyscloud_flow_milestone Data Source - terraform-provider-genesyscloud"
 subcategory: ""
 description: |-
-  Data source for Genesys Cloud Flow Milestone. Select a milestone by name.
+  Genesys Cloud flow milestone data source. Select an flow milestone by name
```

Comment:
Is it select an or select a flow milestone by name.  You might want to run this through grammarly